### PR TITLE
Update fabric crashlytics to firebase crashlytics

### LINKED ISSUE #1553: Update crashlytics SDK version
**Summary:**

Current crashlytics SDK version is about to be depreceated and needs to be updated 

> This version is deprecated. Details: The Fabric Crashlytics SDK is now deprecated and will continue reporting your app's crashes only until November 15, 2020. To continue getting crash reports in the Firebase console, make sure you upgrade to the Firebase Crashlytics SDK version 17.0.0+.  For more information, visit https://firebase.google.com/docs/crashlytics/upgrade-sdk. Consider switching to recommended version 17.2.2. 

**Steps to reproduce:**

Check version in build.gradle of :app module

**Expected behaviour:**

Upgrade version to 17.x.x

**Observed behaviour:**

Currently in 2.6.7

### PR DESCRIPTION
**Pull request by Sparsh1212**
Fixes #1553 

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Sparsh1212** [comment]
@miPlodder @devansh-299 Please review.

**Comment by garvit984** [comment]
@Sparsh1212 Please squash all the commits into a single commit.

**Comment by Sparsh1212** [comment]
> @Sparsh1212 Please squash all the commits into a single commit.

Done!

File: app/build.gradle [hunk 1]
```diff
@@ -117,9 +117,7 @@ dependencies {
     //Butter Knife
     implementation "com.jakewharton:butterknife:$butterKnifeVersion"
     kapt "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
-    implementation('com.crashlytics.sdk.android:crashlytics:2.6.7@aar') {
-        transitive = true
-    }
+    implementation 'com.google.firebase:firebase-crashlytics:17.3.0'
```
Comment (miPlodder) [inline app/build.gradle]: Let's follow the standard practice of defining versions at one place, and then using them.

File: build.gradle [hunk 2]
```diff
@@ -4,11 +4,10 @@ buildscript {
     repositories {
         google()
         jcenter()
-        maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.3.2'
-        classpath 'io.fabric.tools:gradle:1.31.2'
+        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.4.1'
```
Comment (miPlodder) [inline build.gradle]: Let's use a variable from versions, let's do the same for other classpath dependencies also, which are missed as I see.

**Review by miPlodder — CHANGES_REQUESTED** [review]
- Take care of the comments given
- Also, add #issue-id in your commit message, so we can get to know what issue does it resolve. (from commit-msg)

**Comment by Sparsh1212** [comment]
> * Take care of the comments given
> * Also, add #issue-id in your commit message, so we can get to know what issue does it resolve. (from commit-msg)

Please review the updated changes.

**Review by miPlodder — APPROVED** [review]
Looks Good.
Nice job

**Pull request merged**